google/cloud/bigquery/client.py

```diff
@@ -3405,6 +3405,91 @@ def query(
         else:
             raise ValueError(f"Got unexpected value for api_method: {repr(api_method)}")
 
+    def query_and_wait(
+        self,
+        query,
+        job_config: Optional[QueryJobConfig] = None,
+        location: Optional[str] = None,
+        project: Optional[str] = None,
+        retry: retries.Retry = DEFAULT_RETRY,
+        timeout: TimeoutType = DEFAULT_TIMEOUT,
+        job_retry: retries.Retry = DEFAULT_JOB_RETRY,
+        page_size: Optional[int] = None,
+        max_results: Optional[int] = None,
+    ) -> RowIterator:
+        """Run the query, wait for it to finish, and return the results.
```

Comment:
This docstring is more comprehensive than the docstring in `__job_helpers.py`. Is that intentional OR will the two docstrings be revised to provide a comprehensive description of the `query_and_wait()` function in both locations.

Reply:
Good point. Fixed! They now both have the same content. There was some important content not in the public version, so this makes sure that's now documented.